[BUG] Reported crash in FileUploader

Crash reported in Google console

```
java.lang.RuntimeException: 
  at android.app.ActivityThread.handleServiceArgs (ActivityThread.java:4222)
  at android.app.ActivityThread.access$2100 (ActivityThread.java:231)
  at android.app.ActivityThread$H.handleMessage (ActivityThread.java:1984)
  at android.os.Handler.dispatchMessage (Handler.java:107)
  at android.os.Looper.loop (Looper.java:214)
  at android.app.ActivityThread.main (ActivityThread.java:7682)
  at java.lang.reflect.Method.invoke (Native Method)
  at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run (RuntimeInit.java:516)
  at com.android.internal.os.ZygoteInit.main (ZygoteInit.java:950)
Caused by: java.lang.NullPointerException: 
  at com.owncloud.android.files.services.FileUploader.onStartCommand (FileUploader.java:303)
  at android.app.ActivityThread.handleServiceArgs (ActivityThread.java:4204)
```

--- a/owncloudApp/src/main/java/com/owncloud/android/files/services/FileUploader.java
+++ b/owncloudApp/src/main/java/com/owncloud/android/files/services/FileUploader.java
@@ -300,7 +300,7 @@ public class FileUploader extends Service
 
         Account account = intent.getParcelableExtra(KEY_ACCOUNT);
         Timber.d("Account to upload the file to: %s", account);
-        if (!AccountUtils.exists(account.name, getApplicationContext())) {
+        if (account == null || !AccountUtils.exists(account.name, getApplicationContext())) {
             return Service.START_NOT_STICKY;
         }
 

